<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список определенных преподавателей</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <item row="3" column="0">
      <widget class="QComboBox" name="degree">
       <item>
        <property name="text">
         <string>Любая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ассистент</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Профессор</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Доцент</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Старший преподаватель</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>От</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="experience_from"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>До</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="experience_to">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="department"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Кафедра</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Категория</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Стаж работы</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Получить результаты</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
